src/CalcViewModel/CalcViewModel.vcxproj

```diff
@@ -408,6 +408,17 @@
       <Project>{311e866d-8b93-4609-a691-265941fee101}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemDefinitionGroup Condition="!Exists('DataLoaders\DataLoaderConstants.h')">
```

Comment:
It seems like you might be able to simplify this section by using a [`Choose/When/Otherwise`](https://docs.microsoft.com/en-us/visualstudio/msbuild/choose-element-msbuild?view=vs-2019), because then you'd only have to have a single condition check authored for `Condition="!Exists('DataLoaders\DataLoaderConstants.h')`